ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -226,41 +226,54 @@ module Ldap = struct
       try Ok (of_string str)
       with _ -> Error (ldap "invalid value for key '%s'" k)
     in
+    let get_int_with_default of_string ~k ~default =
```

Comment:
I think `k` is not a good name; I assume it stands for `key` - so I would use that instead.